### DISCUSSION TITLE
Hi , whenever i click on the item it will select all items in the dropdown even though i was passing multiEnable false

### DISCUSSION THREAD

**Discussion by shashikumarwml** [srivastavaanurag79/react-native-paper-select#40] [Q&A] [answered]
<PaperSelect
        label="Category"
        value={pickerValue.selectedList}
        onSelection={value => {
          console.log('value: ', value);
          // setPickerValue({
          //   value: value.text,
          //   error: '',
          // });
        }}
        selectedArrayList={[]}
        arrayList={[...pickerValue.list]}
        multiEnable={false}
        selectAllEnable={false}
      />
the code is here

**Comment by srivastavaanurag79** [comment]
Will test it and let you know if there is a bug

**Comment by Peeet93** [comment]
@shashikumarwml you need to use _id as a property, otherwise it selects all

**Comment by shashikumarwml** [comment]
Thanks for your reply Srivasthav

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by srivastavaanurag79** [comment, marked as answer]
Just make sure your default list which you provide must contain a unique `_id` for each element somewhat like this: 

`list: [
      { _id: '1', value: 'MALE' },
      { _id: '2', value: 'FEMALE' },
      { _id: '3', value: 'OTHERS' },
    ],`

For more detail, you can check out the [example](https://anurag-srivastava.gitbook.io/react-native-paper-select/example).
